packages/ui-tree-browser/package.json

```diff
@@ -42,7 +42,7 @@
     "@instructure/ui-testable": "^7.3.1",
     "@instructure/ui-themeable": "^7.3.1",
```

Comment:
If themeable is not used in any of the 3 components, this can be deleted too.

Reply:
Fixed